packages/content-api/lib/index.js

```diff
@@ -90,12 +94,27 @@ export default function GhostContentAPI({url, host, ghostPath = 'ghost', version
             headers
         }).then((res) => {
             if (!Array.isArray(res.data[resourceType])) {
-                return res.data[resourceType];
+                if (formatResponse === true){
+                    return res.data[resourceType];
+                } else {
+                    return res.data;
+                }
+                
             }
             if (res.data[resourceType].length === 1 && !res.data.meta) {
-                return res.data[resourceType][0];
+                if (formatResponse === true){
+                    return res.data[resourceType][0];
+                } else {
+                    return res.data;
+                }
+                
+            }
+            if(formatResponse === true ){
+                return Object.assign(res.dat, {meta: res.data.meta});
```

Comment:
this should be `return Object.assign(res.data, {meta: res.data.meta});` and not `return Object.assign(res.dat, {meta: res.data.meta});`